cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -95,10 +95,10 @@
         {% elif additional_check.type == 'duplicate_statement_id' %}
             <tr>
                 <td>
-                    {% trans 'This statement ID has been used more than once.' %}
+                    {% blocktrans %}<code>statementId</code> value used in multiple statements. Different statements should not have the same <code>statementId</code> value.{% endblocktrans %}
```

Comment:
```suggestion
                    {% blocktrans %}<code>statementId</code> value used in multiple statements. Different statements should not have the same <code>statementId</code> value.{% endblocktrans %}
```

should this be a "must" or is it only a "should"?

Reply:
I'm leaving as a should. (There's an outlying case where a statementId might be re-used: redaction.) I'm not trying to replicate the normative use of 'must' and 'should' from the docs.